guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -22,27 +20,46 @@ ifdef::foreman-el,katello,satellite[]
 [cols="1,1,1",options="header"]
 |====
 |Directory |Installation Size |Runtime Size
+
+|/var/log/ |10 MB |10 GB
+
+|/var/lib/pgsql |100 MB |20 GB
+
+|/usr | 5 GB | Not Applicable
+
+|/opt/puppetlabs | 500 MB | Not Applicable
+
 ifdef::katello,satellite,orcharhino[]
-|/var/lib/pulp |1 MB |300 GB
+|/var/lib/pulp/ |1 MB |300 GB
+
+|/var/lib/qpidd/ |25 MB | Not Applicable
```

Comment:
Small nit - in some cases there is a trailing `/` and in others not, could you align on a single method?

Reply:
Addressed :+1: